data/trustgo.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "TrustGo",
+  "description": "Trusta Labs aims to continuously develop MEDIA Score as the infrastructure for on-chain user reputation assessment. In the KYA product called TrustGo, every user can look up their own unique MEDIA Score on Starknet. It helps to fairly identify accounts that contribute to the Starknet ecosystem. Through MEDIA Score, users can gain a deeper understanding of their on-chain activities and value, while project teams can accurately allocate resources and incentives to users who truly contribute. TrustGo and MEDIA score will accelerate the grow of Starknet ecosystem.",
+  "short_description": "Know Your Account. Track Your Web3 Journey.",
+  "tags": [
+    "Infrastructure"
+  ],
+  "contracts": [],
+  "goerliContracts": [],
+  "audits": [],
+  "dotw": false,
```

Comment:
dotw is now dotm, and this is missing the verified status. Please replace `"dotw": false`, with the following:

```
  "verified": false,
  "dotm": false,
```

Reply:
I have fixed the code issue you pointed out. Thanks!